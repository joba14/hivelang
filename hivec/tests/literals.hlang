
// Description:
//     Testing literals.
// 
// Expectations:
//     The program should produce this output: "50\nHello!\n7\n"

procedure main do
	50 printn
	"Hello!\n" over over 1 1 syscall3 drop
	drop printn
end
