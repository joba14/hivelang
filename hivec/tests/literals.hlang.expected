50
Hello!
7
